apps/projects/app/components/Content/Settings.js

```diff
@@ -421,28 +437,24 @@ Settings.propTypes = {
   onLogin: PropTypes.func.isRequired,
 }
 
-const StyledInputDropDown = ({ children }) => {
+const StyledInputDropDown = ({ children, className }) => {
```

Comment:
I think as a convention, we should not prefix with the "Style" word components that are not Styled components. There is a simple reason. only styled components can be extended by other styled components, if we ever try in the future to extend this component, for example, with styled(StyledNumberFormat) that will cause an error and be confusing for the developer, please, avoid using Styled prefixes for non-styled-components...

Also why don't just use a style component for this one? I don't like to see so much inlined CSS in the components.

Another good reason, if the other 2 are not enough, is that inlined css is not easily lintable, making it error prone, while styled-components are currently being automatically linted in our project config (with stylelint)

Reply:
I've addressed most of these comments in the first spot where you made them, but I'll explain some specifics about `StyledInputDropDown`:

* This component was already non-`styled`. I did not change that; I do not think it adds to the clarity of changes to make such unrelated alterations.
* As I already mentioned, passing `className` allows wrapping this component with `styled`, which is what the inline `css` prop does behind the scenes. This allowed me to move the `margin-bottom` to only the occurrence of `StyledInputDropDown` where we needed it.

Reply:
I have moved this component to the Shared directory and renamed it to `NumberInput`. I used regular `styled` components rather than the inline `css` prop. I also added a comment explaining why the wrapping div is necessary. Let me know what you think, Otto!

Reply:
Thank you for that, anyway, since you are coding a lot of the frontend, I guess it is fair to use whatever convention makes you comfortably working with the code, because I don't consider this enough important to ask you refactoring or renaming components anymore. Sorry if my previous review was a bit out of place.
Your reasons were pretty convincing for me.